webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const packagejson = require('./package.json');
 
-const dashLibraryName = packagejson.name.replace(/-/g, '_');
+const dashLibraryName = packagejson.name.replace('@plotly/','').replace(/-/g, '_');
```

Comment:
This is OK here, but if we propagate this fix to the boilerplate we'll want to remove ANY scope, not just `@plotly`. Something like this:

```suggestion
const dashLibraryName = packagejson.name.replace(/^@[^\/]+\//, '').replace(/-/g, '_');
```